INSTALLATION.md

```diff
@@ -102,12 +102,36 @@ If all goes well, a new ROM should be built at `build/hackeroot-mq/hackeroot-mq.
 
 Both of these have the disadvantage that the ordering of the terminal output is scrambled, so for debugging it is best to stick to one thread (i.e. not pass `-j` or `-jN`).
 
-#### 6. Creating a BPS patch
+#### 6. Other targets
 
+##### Compressing the rom
+To compress the rom, run the following command:
+
+```bash
+make compress
+```
+
+##### Building for Wii VC
+To build a .wad for Wii VC, run the following command:
+
+```bash
+make wad
+```
+
+**NOTE:** this is assuming you followed the instructions for Wii VC in [step 3](#3-prepare-a-base-rom).
+
+##### Creating a BPS patch
 To generate a .bps patch you can run the following command:
 
 ```bash
 make patch
 ```
 
 The patch will be in the ``build/hackeroot-mq/`` folder.
+
+#### 7. Toggle HackerOoT Features
+
+You can customize which HackerOoT features you want in your project and which not in the `include/config/` folder.
+Just define the features as `true` or `false` or, if necessary, put in the value you want (e.g. Text speed).
+
+**TIP:** if you are using VS Code, it might show you errors in your files with red squiglly lines. Those are false alarms, to get rid of them, you can place [THIS FILE](https://gist.github.com/krm01/7345ca80e0e7f6b083a0ea4e85515d4b) by [@kentonm](https://www.github.com/krm01) in your `.vscode` folder (You might need to create it first).
```

Comment:
hackeroot has updated vscode instructions inside docs/vscode.md, also this file explains how to setup the Ares GDB stuff on vscode

Reply:
I'd link that file here then because it's a bit hard to find right now imo

Reply:
yeah I agree it's kinda hidden, would be nice to have something written about it in the readme